src/components/VirtualizedTreeSelect.js

```diff
@@ -352,10 +352,14 @@ const Menu = (props) => {
           props.selectProps.listProps.onScroll(e.target);
```

Comment:
@ledsoft It does not seem to be used anywhere, are there scenarios where it is needed? It seems like the component works better without this custom Menu component

Reply:
Could you be more specific as to what you mean by "works better"?

Reply:
Without custom Menu, expected behavior is working as intended (i.e. keyboard navigation enabled, no scroll to top when clicking). With the custom Menu, I had to implement `onMouseEnter` to enable keyboard navigation (does not work with `onMouseDown`)

Reply:
Ok, in that case why don't you go ahead and remove the custom `Menu` component altogether. It was added there just to fix some scrolling issues, maybe that is no longer necessary.

Reply:
Ok, I was just concerned it is here for other cases I'm not aware of but yes then I'll remove it.